requirements/base.txt

```diff
@@ -1,31 +1,45 @@
-elasticsearch==1.6.0
 Jinja2==2.7.3
-pytz
-feedparser==5.2.1
-unipath>=1.1,<=2.0
-django>=1.8,<=1.8.99
-six==1.9.0
-python-dateutil==2.1
-requests==2.7.0
-requests_toolbelt==0.6.0
-git+https://github.com/rosskarchner/govdelivery.git#egg=govdelivery
-icalendar==3.9.0
-django-localflavor==1.1
-wagtail==1.5.2
-html5lib==0.999999
+MarkupSafe==0.23
 MySQL-python==1.2.5
-django-extensions==1.5.5
+Pillow==3.3.0
+Unidecode==0.4.19
+Willow==0.3.1
+Werkzeug==0.10.4
+beautifulsoup4==4.4.1
 boto==2.38.0
-django-storages==1.1.8
+django-appconf==1.0.2
+django-compressor==2.0
+django-extensions==1.5.5
 django-haystack==2.4.1
-beautifulsoup4==4.4.1
+django-localflavor==1.1
+django-modelcluster==2.0
 django-overextends==0.4.1
 django-reversion==1.10.1
+django-storages==1.1.8
+django-taggit==0.18.3
 django-tinymce==2.2.0
-Wand==0.4.2
+django-treebeard==3.0
+django>=1.8,<=1.8.99
+djangorestframework==3.3.3
+elasticsearch==1.6.0
 git+https://github.com/cfpb/publish_eccu.git#egg=publish_eccu
-argparse==1.3.0
-suds==0.4
+git+https://github.com/rosskarchner/govdelivery.git#egg=govdelivery
+html5lib==0.9999999
+icalendar==3.9.0
 lxml==3.6.0
 pbr==1.9.1
-django-compressor==2.0
\ No newline at end of file
+py==1.4.31
+python-dateutil==2.1
+pytz
```

Comment:
I tried. It seems that pytz has this weird versioning that uses a YYYY.MM.DD format. Pip does not understand how to resolve this and therefore complains about it. I'm not sure what the right solution here is; it was previously left unpinned for, presumably, the same reason. Open to suggestions as to what to do.